tar/tarsnap.1-man.in

```diff
@@ -397,6 +397,10 @@ Allow descent into synthetic filesystems such as procfs.
 Normally archiving of such filesystems is a silly thing to do, hence the
 name of the option.
 .TP
+\fB\--iso-dates\fP
+(-t mode only)
+Prints file and directory dates as yyyy-mm-dd hh:mm:ss.
```

Comment:
whoosh?  Those were all options that I added.  :)

I mean, ok, you were "inconsistent" in not noticing (some of) my mistakes for 3 or 4 edits I've made to the man pages.  But yeah, I'll look for other such mistakes after I've fixed my own.